Library/Formula/h2object.rb

```diff
@@ -0,0 +1,12 @@
+require 'formula'
+
+class H2object < Formula
+  homepage 'http://h2object.io'
+  url 'http://dl.h2object.io/h2object/macosx/h2object-0.0.1-beta.tar.gz'
```

Comment:
Is there a stable release we can use? It’ll need to go to [Homebrew/homebrew-devel-only](https://github.com/homebrew/homebrew-devel-only) if there is only a beta release.